server/container/guice/guice-common/src/test/java/org/apache/james/AbstractJmapJamesServerTest.java

```diff
@@ -39,15 +40,15 @@
 import com.jayway.restassured.builder.RequestSpecBuilder;
 import com.jayway.restassured.http.ContentType;
 
-public abstract class AbstractJmapJamesServerTest {
+public abstract class AbstractJmapJamesServerTest<T extends GuiceJamesServer & JmapServer & WebAdminServer> {
```

Comment:
What about really defining T as an interface extending these ones? I'm not sure you really need generics here.